src/site/content/en/progressive-web-apps/using-a-pwa-in-your-android-app/index.md

```diff
@@ -63,8 +63,12 @@ nightly version.
 ### Quality Criteria
 
 Web developers should use a Trusted Web Activity when they want to include web content in an
-Android app.  Web content in a Trusted Web Activity must meet Lighthouse's PWA install quality criteria and
-additional Android-specific criteria such as [policy compliance][9].
+Android app. Web content in a Trusted Web Activity must meet Lighthouse's PWA install quality
+criteria and additional Android-specific criteria such as [policy compliance][9].
+
+**Important:** When targeting an application for users under 13 years old on the Play Store
+additional [Play Family policies][18] apply, which may be incompatible with using Trusted Web
+Activity.
```

Comment:
```suggestion
{% Aside 'caution' %}
  When your app is designed primarily for children under 13,
  additional [Play Family policies][18] apply, which may be incompatible
  with using Trusted Web Activity.
{% endAside %}
```

Minor text tweak and using the [Caution widget](https://web.dev/handbook/web-dev-components/#caution-asides).